src/main/java/org/jenkinsci/test/acceptance/utils/IOUtil.java

```diff
@@ -33,6 +35,8 @@
 
 public class IOUtil {
 
+    private static final int CONNECTION_TIMEOUT = Integer.parseInt(SystemEnvironmentVariables.getPropertyVariableOrEnvironment("CONNECTION_TIMEOUT", "10000"));
```

Comment:
No idea about the env variable naming convention, but I've seen [here](https://wiki.jenkins.io/display/JENKINS/Features+controlled+by+system+properties) it's CamelCase based for the Jenkins Core. Is this the case?

Reply:
As I could see in ATH code, all environment variables are named using a upper case word (or upper case words separated by `_`)